arch/arm/cpu/armv8/start.S

```diff
@@ -16,19 +16,27 @@
  *
  *************************************************************************/
 
-.globl	_start
-_start:
 #if defined(CONFIG_LINUX_KERNEL_IMAGE_HEADER)
+
 #include <asm/boot0-linux-kernel-header.h>
+
 #elif defined(CONFIG_ENABLE_ARM_SOC_BOOT0_HOOK)
+
 /*
  * Various SoCs need something special and SoC-specific up front in
  * order to boot, allow them to set that in their boot0.h file and then
  * use it here.
  */
+
+.globl	_start
```

Comment:
`_start:` missing? I checked couple of "boot0.h" files, none of them define `_start` symbol

Reply:
Hi @lorc, I've checked them all.

|File link|Whether it has _start defined?|
|---|---|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-imx8/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-px30/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-sunxi/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3036/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3128/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3188/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk322x/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3288/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3308/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3328/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3368/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rk3399/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rv1108/boot0.h|No|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-bcm235xx/boot0.h|Yes|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-bcm281xx/boot0.h|Yes|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/include/asm/arch-rockchip/boot0.h|Yes|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/mach-rmobile/include/mach/boot0.h|Yes|
|https://github.com/xen-troops/u-boot/blob/master/arch/arm/mach-socfpga/include/mach/boot0.h|Yes|

Reply:
Thanks. But why this change is in "Add xen-specific targets to the BAZEL build" commit?

For me it looks like something that should be in a separate commit, probably worth upstreaming.

Reply:
Hi @lorc, it is in the same commit, because the Bazel build is failing without this change. As we agreed - each commit should be compilable. That motivated me to place it in such a way. The thing I can do is place the commit with the compilation issue fix before the "Add xen-specific targets to the BAZEL build". That will ensure atomic, still buildable commits. I will do that tomorrow.

Reply:
Yep, this is the right thing to do.

Reply:
Hi @lorc, done.